src/App.tsx

```diff
@@ -41,22 +45,25 @@ const useStyles = makeStyles(theme => ({
 
 const App: React.FC = () => {
   const classes = useStyles();
+  const [settingsModalOn, setSettingsModalOn] = useState(false);
+  const handleSettingsSubmit = (selectedVideoDevice: MediaDeviceInfo | undefined, selectedAudioDevice: MediaDeviceInfo | undefined): void => {
+    setAudioAndVideoDevice(selectedAudioDevice, selectedVideoDevice);
+    toggleSettings();
```

Comment:
For my understanding, we are setting the audio /video device, and then which settings are being toggled?

Reply:
the `toggleSettings` refers to the settings modal itself!

Reply:
Thank you!